core/kazoo_stdlib/src/kz_json.erl

```diff
@@ -324,6 +324,8 @@ merge_left(_K, {'right', ?JSON_WRAPPER(_)=Right}) ->
     {'ok', merge(fun merge_left/2, Right, new())};
 merge_left(_K, {'right', V}) -> {'ok', V};
 
+merge_left(_K, {'both', ?EMPTY_JSON_OBJECT=Left, ?JSON_WRAPPER(_)=_Right}) ->
+    {'ok', Left};
```

Comment:
@jamesaimonetti shouldn't this be `{'ok', Right}` ? 
we're merging the `Left` object into the `Right`, the resulting should be `Right` overridden with `Left` values.